server/integration/endpoint/src/main/java/org/infinispan/server/endpoint/deployments/AbstractCacheStoreExtensionProcessor.java

```diff
@@ -0,0 +1,52 @@
+package org.infinispan.server.endpoint.deployments;
+
+import org.infinispan.server.endpoint.Constants;
+import org.infinispan.server.endpoint.subsystem.ExtensionManagerService;
+import org.jboss.as.server.deployment.DeploymentPhaseContext;
+import org.jboss.msc.service.Service;
+import org.jboss.msc.service.ServiceBuilder;
+import org.jboss.msc.service.ServiceController;
+import org.jboss.msc.service.ServiceName;
+import org.jboss.msc.value.InjectedValue;
+
+import static org.infinispan.server.endpoint.EndpointLogger.ROOT_LOGGER;
+
+public abstract class AbstractCacheStoreExtensionProcessor<T> extends AbstractServerExtensionProcessor<T> {
+
```

Comment:
I must have messed something when recompiling. After running a clean build everything works properly. Sorry for confusion.